src/components/__tests__/useResource.test.tsx

```diff
@@ -0,0 +1,43 @@
+import { Mock } from 'firebase-nightlight';
```

Comment:
Do we want to officially move all tests to this `__tests__` folder? I'm used to having them exist with the components, but usually I follow the convention of also having a folder for component anyways 👍 Either way works as long as we have a standard.

Reply:
yeah I was gonna put up a PR to do that.. I think it makes sense as we get more files. I don't have a strong opinion either way.